tests/test_rebar.py

```diff
@@ -248,6 +248,66 @@ def get_me():
         )
         self.assertEqual(resp.status_code, 400)
 
+    def test_delete_content_type_default_json(self):
+        rebar = Rebar()
+        registry = rebar.create_handler_registry()
+
+        # Test DELETE with None return schema
+        @registry.handles(
+            rule="/me",
+            method="DELETE",
+            marshal_schema={204: None},
+        )
+        def delete_me():
+            return None, 204
+
+        # Test DELETE with non-None return schema
+        @registry.handles(
+            rule="/me2",
+            method="DELETE",
+            marshal_schema={204: MeSchema()},
+        )
+        def delete_me2():
+            return {"user_name": ""}, 204
+
+        app = create_rebar_app(rebar)
+
+        resp = app.test_client().delete(path="/me")
+        self.assertEqual(resp.status_code, 204)
+        self.assertEqual(resp.data.decode("utf-8"), "")
+        self.assertEqual(
+            next(header[1] for header in resp.headers.to_list() if header[0] == "Content-Type"),
+            "application/json"
+        )
+
+        resp = app.test_client().delete(path="/me2")
+        self.assertEqual(resp.status_code, 204)
+        self.assertEqual(resp.data.decode("utf-8"), "")
+        self.assertEqual(
+            next(header[1] for header in resp.headers.to_list() if header[0] == "Content-Type"),
```

Comment:
As above. e.g. https://werkzeug.palletsprojects.com/en/0.14.x/quickstart/ includes the example
```python
>>> response.headers['content-type']
'text/plain; charset=utf-8'
```